### DISCUSSION TITLE
[v7] Please bring back step hold of NumberInput from Mantine v6

### DISCUSSION THREAD

**Discussion by medihack** [mantinedev/mantine#5292] [Feature requests]
The NumberInput in Mantine v6 had the convenient feature of continuously increasing or decreasing the value when holding one of the control buttons. This could even be customized with `stepHoldInterval` and `stepHoldDelay`. Our users (medical reporting software) used this feature a lot. But now it seems to be gone in Mantine v7. Can this feature be brought back? The only workaround I see is to use custom controls otherwise, but IMHO it was a great feature of the NumberInput.

**Comment by rtivital** [comment]
It is planned to be added back in 7.5
